src/components/ui-shell/_platform-nav.scss

```diff
@@ -0,0 +1,146 @@
+  }
+
+  //----------------------------------------------------------------------------
+  // Platform Nav Section
+  //----------------------------------------------------------------------------
+  .#{$prefix}--platform__nav-section:not(:last-child)::after {
+    display: block;
+    content: '';
+    height: 1px;
+    background-color: $ibm-colors__gray--80;
+    margin: 0 get-fixed-size(2);
+  }
+
+  //----------------------------------------------------------------------------
+  // Platform Nav Item
+  //----------------------------------------------------------------------------
+  .#{$prefix}--platform__nav-item {
+    display: flex;
+    align-items: center;
+  }
+
+  //----------------------------------------------------------------------------
+  // Platform Nav Link
+  //----------------------------------------------------------------------------
+  a.#{$prefix}--platform__nav-link {
+    display: flex;
+    align-items: center;
+    color: $ibm-colors__gray--10;
+    text-decoration: none;
+    font-size: rem(14px);
+    font-weight: 400;
+    width: 100%;
+  }
+
+  a.#{$prefix}--platform__nav-link:hover {
+    background-color: $ibm-colors__gray--80;
+    color: $ibm-colors__gray--10;
+  }
+
+  a.#{$prefix}--platform__nav-link:focus {
+    outline: get-fixed-size(0.5) solid $ibm-colors__blue--60;
+    outline-offset: get-fixed-size(-0.5);
+  }
+
+  //----------------------------------------------------------------------------
+  // Platform Nav Category
+  //----------------------------------------------------------------------------
+  .#{$prefix}--platform__category {
+    width: 100%;
+  }
+
+  .#{$prefix}--platform__category-toggle {
+    @include button-reset($width: true);
+
+    display: flex;
+    align-items: center;
+  }
+
+  .#{$prefix}--platform__category-toggle:hover {
+    background-color: $ibm-colors__gray--80;
+  }
+
+  .#{$prefix}--platform__category-toggle:focus {
+    outline: get-fixed-size(0.5) solid $ibm-colors__blue--60;
+    outline-offset: get-fixed-size(-0.5);
+  }
+
+  .#{$prefix}--platform__category-title {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-right: get-fixed-size(2);
+    width: 100%;
+    color: $ibm-colors__gray--10;
+    font-size: rem(14px);
+    font-weight: 400;
+  }
+
+  .#{$prefix}--platform__category-items {
+    display: none;
+    visibility: hidden;
+  }
+
+  .#{$prefix}--platform__category-item > a.#{$prefix}--platform__nav-link {
+    display: flex;
+    align-items: center;
+    min-height: get-fixed-size(5);
+    padding-left: get-fixed-size(7);
+  }
+
+  .#{$prefix}--platform__category--expanded .#{$prefix}--platform__category-title {
+    font-weight: 600;
+  }
+
+  .#{$prefix}--platform__category--expanded .#{$prefix}--platform__category-title > svg {
+    transform: rotate(180deg);
+  }
+
+  .#{$prefix}--platform__category--expanded .#{$prefix}--platform__category-items {
+    display: block;
+    visibility: visible;
```

Comment:
[`visibility:visible` is a tricky thing](https://jakearchibald.com/2014/visible-undoes-hidden/) - `visibility:inherit` yields more intuitive result.